lib/hadethDetails/HadethDetailsScreen.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+import 'package:islami_c9_mon/home/hadeth/Hadeth.dart';
+
+class HadethDetailsScreen extends StatelessWidget {
+  static const String routeName = 'hadeth-details';
+
+  const HadethDetailsScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    var args = ModalRoute.of(context)?.settings.arguments as Hadeth;
+    return Container(
+        decoration: const BoxDecoration(
+            image: DecorationImage(
+                image: AssetImage('assets/images/main_background.png'),
+                fit: BoxFit.fill)),
```

Comment:
please move this decoration to a separate widget because we are using it many screens